### ISSUE TITLE
Duplicate line in nzbget.conf

### ISSUE THREAD

**Issue by alienx2** [nzbget/nzbget#170]
Can you fix bug? theres always giving error bec i saw in nzbget.conf there is alot of lines are duplicated every save settings. latest version is 16.4

**Comment by alienx2** [comment]
and here error: 

nzbget.conf(1934): Invalid value for option "Feed1.PauseNzb": ""
nzbget.conf(1932): Invalid value for option "Feed1.Backlog": ""

**Comment by hugbug** [comment]
That's not a bug. You configuration is invalid. Check that option ConfigTemplate points to the default (not altered) configuration template file (not to your config file), which comes with nzbget.

If you install using official installer packages (Windows, Linux, OS X), everything is configured correctly.
If you install using third-party packages they may be misconfigured.

To fix the error either use official packages or fix your install manually by downloading source code of your nzbget version, extracting file nzbget.conf into any location and settting option ConfigTemplate to the extracted file. You may also need to manually cleanup your config file from errorneous entries.

**Comment by alienx2** [comment]
i did removed Feed1.PauseNzb but repeat again it was wrote appended in nzbget.conf from nzbget after i click save settings in nzbget. look this picture:
![bug](https://cloud.githubusercontent.com/assets/1681482/13029228/ec321f74-d2c0-11e5-8f42-b9a80c4dfded.jpg)

**Comment by hugbug** [comment]
Please read my whole previous post.

**Comment by alienx2** [comment]
it was installed via entware-ng package (after compiled by one of creator entware-ng got from this github). not direct from thisgithub.

**Comment by hugbug** [comment]
> To fix the error either use official packages or fix your install manually by downloading source code of your nzbget version, extracting file nzbget.conf into any location and settting option ConfigTemplate to the extracted file. You may also need to manually cleanup your config file from errorneous entries.

**Comment by hugbug** [comment]
Assuming the issue is fixed.
If you need further assistance please write back.

**Comment by jeroenh** [comment]
I am experiencing the exact same issue with version 16.4.
I've just cleaned up my nzbget.conf because it was getting ridiculously long (>800KB).

When I go to settings and press "Save all changes" new duplications are introduced. 
It seems that the values that have been changed from the original file are duplicated at the end of the file. For me, all values that I have created in the Server, Category and Task settings are duplicated.

Furthermore, the duplication is not accurate: empty values are introduced, creating errors such as described above. I specifically see that for Task.Command, Server.Active, Server.JoinGroup, Server.Encryption and Category.Unpack.
